PackagePrice.java

```diff
@@ -0,0 +1,52 @@
+package pricecomparison;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class PackagePrice {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		WebDriver driver = new FirefoxDriver();
+
+		String appUrl = "http://www.tripfactory.com/package/get-price-req?pkgId=27433&ahdate=06%2F05%2F2016";
+		driver.get(appUrl);
+
+		WebElement wE = driver.findElement(By.xpath(".//*[@id='invCalMth2']/table"));
+
+		System.out.println(wE.getText());
+
+		driver.findElement(By.xpath(".//*[@id='invCalMth2']/table/tbody/tr[2]/td[1]/div[2]/div[3]")).click();

```

Comment:
But wht you are picking on line 21, is a child of what you are picking on 17.
The major flaw in your code throughout is, not picking all elements as a list, but picking a single element at a time.

Please work on understanding how you can use xpath to select multiple elements at a time. Its time to stop using the xpath returned by firebug, as that will give you xpath only to one element. Your goal should be to capture a set of elements with similar properties (like shared classes example shared with you for selecting all calendar prics and dates as a list, and not webelement as in Line 17.